superset/reports/commands/create.py

```diff
@@ -113,3 +114,20 @@ def validate(self) -> None:
             exception = ReportScheduleInvalidError()
             exception.add_list(exceptions)
             raise exception
+
+    def _validate_report_extra(self, exceptions: List[ValidationError]) -> None:
+        extra = self._properties.get("extra")
+        dashboard = self._properties.get("dashboard")
+
+        if extra is None or dashboard is None:
+            return
+
+        dashboard_tab_ids = extra.get("dashboard_tab_ids")
+        if dashboard_tab_ids is None:
+            return
+        position_data = json.loads(dashboard.position_json)
+        invalid_tab_ids = [
+            tab_id for tab_id in dashboard_tab_ids if tab_id not in position_data
+        ]
+        if invalid_tab_ids:
+            exceptions.append(ValidationError("Invalid tab IDs selected", "extra"))
```

Comment:
should we append the invalid tab ids to the error message? If not, then we don't need to find all of them, and can instead just check if dashboard_tab_ids is a subset of position_data. Like this:
```python
if not set(dashboard_tab_ids).issubset.(set(position_data.keys())):
  raise Exception
```

Reply:
i think let owners know error details is important. i added extra filed in error message.